source/book/include/marlin/book/Selection.h

```diff
@@ -25,6 +28,40 @@ namespace marlin {
     class Selection {
 
     public:
+      class Hit {
+        friend Selection ;
+        
+      public:
+        // constructor
+        explicit Hit( const std::shared_ptr< const Entry > &entry ) ;
+        // constructor
+        explicit Hit( const std::shared_ptr< Entry > &entry ) ;
+
+        /**
+         *  @brief check if Hit is usable.
+         *  @return false when Entry referenced by Hit no longer exist
+         */
+        [[nodiscard]] bool valid() const ;
+
+        /**
+         *  @brief get key from Entry.
+         */
+        [[nodiscard]] const EntryKey &key() const ;
+
+        /**
+         *  @brief bind Entry to new Handle, for further usage.
+         *  @attention don't use old Handle to the Entry after this.
+         */
+        template <typename T>
+        Handle<Manager<T>> bind() const {
```

Comment:
`handle()` instead or `bind()`. Bind refers to something like std::bind

Reply:
Also it is a weak Entry and bind or lock is a more commen term?